src/nvim/quickfix.c

```diff
@@ -4016,7 +4016,12 @@ static list_T *call_qftf_func(qf_list_T *qfl,
   }
   if (qftf != NULL && *qftf != NUL) {
     typval_T args[1];
-    dict_T *const dict = tv_dict_alloc();
+    dict_T *dict;
+
+    // create the dict argument
+    if ((dict = tv_dict_alloc_lock(VAR_FIXED)) == NULL) {
```

Comment:
`tv_dict_alloc` does not return `NULL` so  `tv_dict_alloc_lock` does not return `NULL.
Forgot to put function attributes.